### PR TITLE
PMM-10513 redact user creds in server url

### PR DESCRIPTION
**Pull request by atymchuk**
PMM-10513

Build: SUBMODULES-2723

### CODE REVIEW

File: build/packages/rpm/server/SPECS/victoriametrics.spec [hunk 1]
```diff
@@ -1,3 +1,5 @@
+%undefine _missing_build_ids_terminate_build
```
Comment (michal-kralik) [inline build/packages/rpm/server/SPECS/victoriametrics.spec:1]: Is this change related to the PR?

**Review by michal-kralik — APPROVED** [review]
All looks good. Just one comment. I'm not sure if the `rpm` change is related to the PR.

File: build/packages/rpm/server/SPECS/victoriametrics.spec (hunk 1 above)
Comment (atymchuk) [reply build/packages/rpm/server/SPECS/victoriametrics.spec:1]: Yes and no. If you look at the logs, the pipeline was failing without this. It's the last component that does not have this statement, all others do already.

**Review by BupycHuk — APPROVED** [review]
